src/storage/kv/cursor.rs

```diff
@@ -483,6 +483,12 @@ impl<'a, S: 'a + Snapshot> CursorBuilder<'a, S> {
             None
         };
         let mut iter_opt = IterOption::new(l_bound, u_bound, self.fill_cache);
+        if let Some(ts) = self.hint_min_ts {
```

Comment:
Can we add some test cases for the builder?

Reply:
It's hard. The filter is deep into rocksdb options and is quite difficult to dig out to check. :(